### PR TITLE
Add Docker image generation.

### PR DESCRIPTION
**Pull request by JonatanAntoni**

### CODE REVIEW

**Comment by spcaipers-arm** [comment]
LGTM, we just need to add the info into the README.md. 
Infos:
 * How to pull the image
 * How to use it in a CLI

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -57,3 +64,56 @@ jobs:
           user: __token__
           password: ${{ secrets.TEST_PYPI_API_TOKEN }}
           verbose: true
+
+  docker:
+    name: Build Docker image
+    needs: update
+    runs-on: 'ubuntu-latest'
```
[the remark this answers is not in the document]
Comment (JonatanAntoni) [reply .github/workflows/release.yml:71]: The `runs-on: ubuntu-latest` here doesn't refer to anything Docker-based. It's the GitHub-hosted virtual machine the workflow to actually build the Docker image runs on.

File: README.rst [hunk 2]
```diff
@@ -165,6 +176,34 @@ Execute command
...
+    AWS_KEEP_EC2_INSTANCES
+    AWS_KEY_NAME
+    AWS_INSTANCE_TYPE
+    AWS_INSTANCE_NAME
+
+This environment file is used to forward the local environment variables into
+the Docker container. Having this prepared one can run ``avhclient`` in a
+container as follows::
+
+    docker run --rm -i --env-file ./env.txt -v $(pwd):/workspace -w /workspace ghcr.io/arm-software/avhclient avhclient [..]
```
Comment (spcaipers-arm) [inline README.rst:203]: If you can add the ~/.aws example here would be great.

**Review by spcaipers-arm — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (DarwinJS) [reply .github/workflows/release.yml:71]: No problem - I am the one misunderstanding - haven't review many GitHub CI configurations as yet.

File: README.rst (hunk 2 above)
Comment (DarwinJS) [reply README.rst:203]: Here is a powershell and bash compatible volume map for AWS creds and ssh keys:

`-v $HOME/.ssh:/root/.ssh -v $HOME/.aws:/root/.aws`

**Review by spcaipers-arm — APPROVED** [review]

**Pull request merged**